book/index.md

```diff
@@ -1,23 +1,14 @@
 # QIIME 2 Cancer AutoFMT Tutorial
 
 The **Q**IIME **2** **C**ancer **A**utoFMT **T**utorial Jupyter **Book** (q2-cat-book) was originally developed for the 2024 [ITN](https://www.itcrtraining.org/) / [QIIME 2](https://qiime2.org)
-Workshop, taught 27 - 29 August 2024 at the National Institutes of Health.
+Workshop, taught 27 - 29 August 2024 at the National Institutes of Health. Now it is the official tutorial for q2-fmt. If you would like to follow along with the online workshop please visit: https://q2-cat-book.readthedocs.io/en/2024.08.27-q2-itn/
```

Comment:
It's convenient for review if you format your markdown files with one sentence per line. Just noting this here - if could you update in other sections when you're making other changes? 

```suggestion
Workshop, taught 27 - 29 August 2024 at the National Institutes of Health.
Now it is the official tutorial for q2-fmt.
If you would like to follow along with the online workshop please visit: https://q2-cat-book.readthedocs.io/en/2024.08.27-q2-itn/
```